frontend/src/services/StorageClassService.ts

```diff
@@ -0,0 +1,16 @@
+import axios from '~/utilities/axios';
+import { StorageClassConfig } from '~/k8sTypes';
+import { ResponseStatus } from '~/types';
+
+export const updateStorageClassMetadata = (
```

Comment:
Maybe this should be called `updateStorageClassConfig` since it only updates the config and at that point metadata could be implied.